Tribler/Core/Modules/restapi/util.py

```diff
@@ -62,6 +96,9 @@ def convert_db_torrent_to_json(torrent, include_rel_score=False):
                 "category": torrent[4], "num_seeders": torrent[5] or 0, "num_leechers": torrent[6] or 0,
                 "last_tracker_check": torrent[7] or 0}
 
+    if len(torrent) >= 11:
```

Comment:
This whole mechanism with these "magic tuples" is so fragile but it will be removed when we can get rid of Dispersy fortunately 👍

Reply:
exactly!